buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -1068,13 +1043,6 @@ class BuildPlugin implements Plugin<Project> {
         return 'auto';
```

Comment:
Yes, I actually wasn't using this method at all and instead was naively using `max-workers` but I've updated this such that we fall back to `Runtime.availableProcessors()` instead of returning "auto".

Reply:
We should keep the default in the randomized runner that capped that to 4, or go with half the processors, using all of them is known to cause tests to fail.

Reply:
Done.